app/scripts/modules/core/src/managed/ManagedResourceDetailsIndicator.css

```diff
@@ -0,0 +1,20 @@
+.ManagedResourceDetailsIndicator {
+  background-color: var(--color-mineshaft);
+  color: var(--color-white);
+}
+
+.ManagedResourceDetailsIndicator .md-logo {
+  background-color: var(--color-black);
+  padding: 18px;
+}
+
+.ManagedResourceDetailsIndicator .summary-message {
+  font-size: 12px;
+  font-weight: 300;
```

Comment:
@gcomstock I made the judgement call to drop this down to 300 because it felt like the text was throwing off the balance of the button/logo a bit — more than happy to go back on that if you prefer it at a normal weight or are worried about contrast (also only have a high-DPI display to judge with). Here's what normal weight looks like for comparison with the images in the PR description:

<img width="344" alt="Screen Shot 2019-11-07 at 9 22 03 PM" src="https://user-images.githubusercontent.com/1850998/68451726-b0299180-01a4-11ea-95d9-8542b435a764.png">

Reply:
Good call. This banner fits the space well.